daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -1,21 +1,30 @@
 ---
 type: docs
-title: "Metadata API reference"
+title: "Sidecar Metadata API reference"
 linkTitle: "Metadata API"
 description: "Detailed documentation on the Metadata API"
 weight: 800
 ---
 
-Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present).
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present). Each loaded components also has information about supported features in the form of component's capabilites.
 
-The Dapr metadata API also allows you to store additional information in the format of key-value pairs.
+These features are only available for State store, Pub/Sub and Binding components. For example, below table shows building blocks and their capabilites. This list might grow in future and only represents the current supported their capabilites of loaded components - 
```

Comment:
```suggestion
## Components
Each loaded component provides its name, type and version and also information about supported features in the form of component capabilities. The table below shows the returned capabilities for component types.

These features are only available for State store, Pub/Sub and Binding components. For example, below table shows building blocks and their capabilites. This list might grow in future and only represents the current supported their capabilites of loaded components - 
```

Reply:
yes, We should put "for a given version" here.
A new version of any component means it has to be registered as a new component(example redis/v2), so I believe features and properties might change between different versions.